<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Play!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add money</string>
   </property>
  </widget>
  <widget class="QPushButton" name="info_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View player info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_rank_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ranking</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome to BlackJack</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdraw_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
